test/package.json

```diff
@@ -21,8 +21,8 @@
     "clean:regression": "rm -rf artifacts/regressionScreenshots",
     "lint": "tslint -c tslint.json visreg/",
     "pretest": "npm run build",
-    "test": "npm run test:regression && npm run test:visreg",
-    "test:regression": "ava node built/regression/generateRegressionTests.js --verbose --serial",
+    "test": "npm run test:regression",
```

Comment:
Check if this is a push or a pr job in travis and run just linting and other sanity checks on push.

Reply:
This created https://jira.rax.io/browse/SURF-1084